dev/reference/mysql-compatibility.md

```diff
@@ -149,3 +149,25 @@ TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
         - For MySQL 5.7: `OFF`
         - For MySQL 8.0: `ON`
     
+### Date and Time
+
+#### Named timezone
+
+TiDB supports named timezones such as `America/Los_Angeles` without having to load the [time zone information tables](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation) as in MySQL.
+
+Because they are built-in, named time zones in TiDB might behave slightly differently to MySQL, and cannot be modified. For example, in TiDB the names are case-sensitive [#8087](https://github.com/pingcap/tidb/issues/8087).
+
+> **Note:**
+>
+> TiKV calculates time-related expressions that can be pushed down to it. This calculation uses the built-in time zone rule and does not depend on the time zone rule installed in the system. If the time zone rule installed in the system does not match the version of the built-in time zone rule in TiKV, the time data that can be inserted cannot be read in a few cases.
```

Comment:
To be more precise and align with "Zero month and zero day", we can change all occurrences of "cannot be read" to "result in a statement error".

Reply:
The rest is cool!